client/packages/system/src/Patient/CreatePatientModal/PatientResultsTab.tsx

```diff
@@ -103,6 +114,9 @@ export const PatientResultsTab: FC<PatientPanel> = ({ patient, value }) => {
         data={data?.map(node => node.patient)}
         columns={columns}
         noDataMessage={t('messages.no-matching-patients')}
+        onRowClick={row => {
```

Comment:
in using this - it was unclear what would happen when you click on the row.
I've put together an example of how a tooltip could be applied.. will make an issue to show that.